src/v2/models/AppState.js

```diff
@@ -106,6 +106,41 @@ export default Model.extend({
     return currentViewState;
   },
 
+  /**
+   * Returns the displayName of the authenticator
+   * @returns {string}
+   */
+  getAuthenticatorDisplayName () {
```

Comment:
Just to add a note - this is currently only used by the IdP and Symantec authenticators. As more authenticators conform this to pattern (conditionally load enroll or verify views) - we'll be able to utilize this more often!

Reply:
Thank you! I think i can use this for on-prem 😄 

Can we do this a little differently? I don't think we need to check challenge, but instead could do
`return currentAuthenticator.displayName || currentAuthenticatorEnrollment.displayName;`
Since displayName will be on either of those objects. 

Alternatively we could get the current view state `getCurrentViewState` and get `relatesTo` from there. relatesTo will always point to the right object. But that might be too much since we already have currentAuthenticator and currentAuthenticatorEnrollment.

Reply:
This simplifies the logic a lot! I guess this level of granularity is just extra noise 😄